pkg/virt-handler/vm.go

```diff
@@ -570,6 +571,10 @@ func (d *VirtualMachineController) setupNetwork(vmi *v1.VirtualMachineInstance,
 	})
 }
 
+func (d *VirtualMachineController) hotUnplugVMIInterfaces(vmi *v1.VirtualMachineInstance) error {
```

Comment:
I'm not sure I understand the point of this function, it seems redundant to me. Is there a reason you cannot call d.netconf.HotUnplugInterfaces(vmi) directly (at line 2976).